video2dataset/subsamplers/clipping_subsampler.py

```diff
@@ -13,19 +13,47 @@
 
 
 def get_seconds(t):
+    """
+    Converts a given time string or float into seconds.
+
+    Args:
+    t (str or float): A time string in the format "%H:%M:%S.%f" or a float representing time in seconds.
+
+    Returns:
+    float: The total number of seconds represented by the input time.
+
+    Note:
+    The current implementation assumes the time string is in the format "%H:%M:%S.%f".
+    To support other formats, consider parameterizing the time_format variable.
+    """
```

Comment:
this function is rather simple and speaks for itself, if you want lint to not complain about a helper function add a leading _. So yeah maybe call this func _get_seconds(t)